src/components/ClassEnrollmentConfirm.tsx

```diff
@@ -0,0 +1,406 @@
+import React, { SetStateAction } from "react";
+import {
+    Box,
+    Button,
+    HStack,
+    VStack,
+    Heading,
+    Text,
+    Stack,
+    Flex,
+} from "@chakra-ui/react";
+import { BackButton } from "@components/BackButton";
+import { CloseButton } from "@components/CloseButton";
+
+type ClassEnrollmentConfirmationProps = {
+    studentData: any; // Not actually any but I don't want to type the entire student object, let me know if I have to do this or if it's done somewhere else that can be used here
+    parentData: any; // ^^^ Same idea
+    pageNum: number;
+    setPageNum: React.Dispatch<SetStateAction<number>>;
+};
+
+const therapyMapping = {
+    PHYSIO: "Physiotherapy",
+    SPEECH_LANG: "Speech and language therapy",
+    OCCUPATIONAL: "Occupational therapy",
+    COUNSELLING: "Counseling",
+    ART: "Art therapy",
+};
+
+export const ClassEnrollmentConfirmation = (
+    props: ClassEnrollmentConfirmationProps,
+): JSX.Element => {
+    return (
+        <Box>
+            <Flex justifyContent="space-between" marginBottom="39px">
+                <BackButton
+                    onClick={() => {
+                        props.setPageNum(props.pageNum - 1);
+                    }}
+                />
+                {/* navigate to browse programs page instead of going back */}
+                <CloseButton href="/" />
+            </Flex>
+            <VStack spacing="45px" alignItems="flex-start">
+                <HStack
+                    alignSelf="flex-start"
+                    mt="15px"
+                    fontWeight="700"
+                    fontSize="36px"
+                >
+                    <Heading>
+                        Confirm Personal Information{" "}
+                        <Heading color="#0C53A0" as="span">
```

Comment:
Please import `colourTheme` and use `colourTheme.colors.Blue` instead, one source of truth.